presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -383,6 +383,88 @@ or credentials for a specific use case (e.g., bucket/user specific credentials).
 This Hadoop configuration property must be set in the Hadoop configuration
 files referenced by the ``hive.config.resources`` Hive connector property.
 
+AWS Security Mapping
+^^^^^^^^^^^^^^^^^^^^
+
+Presto supports flexible security mapping for AWS Lake Formation, AWS Glue, and AWS S3 API calls, allowing for separate
```

Comment:
It's not a generic term, we named it like that since it maps different Presto User Identities to AWS Credentials. Let me know if you have any ideas on the naming.

Reply:
I think you just need to say that in the docs and the code then.

Reply:
The current explanation here on line 389-390: "... allowing for separate credentials or IAM roles for specific users." do you think we should rephrase this better? We tried to explain what we mean by security mapping here.

Reply:
There are more details on what each mapping should include in the following lines

Reply:
But you still never say what a security mapping *is*. You need to define that. Easiest way might be to rewrite the first paragraph without using the phrase "security mapping".

Reply:
Sure, I will rephrase it.